test/test_mesh.py

```diff
@@ -120,3 +120,10 @@ def test_meshtags_from_xdmf(tmp_path, mesh):
     assert volume_meshtags.values.all() == my_model.volume_meshtags.values.all()
     assert facet_meshtags.dim == my_model.facet_meshtags.dim
     assert facet_meshtags.values.all() == my_model.facet_meshtags.values.all()
+
+
+def test_mesh_vertices_from_list():
+    """Check that giving vertices as a list is correctly processed"""
```

Comment:
"correctly processed" isn't very explicit.

Here, you are specifically checking that the `vertices` attributes has the correct type `np.ndarray`